engine/src/main/resources/application.conf

```diff
@@ -68,85 +61,121 @@ call-caching {
-  // For instance data localization and delocalization in GoogleCloudStorage will be done on behalf of another entity (typically a user).
-  // This allows cromwell to localize data that wouldn't be accessible using whatever "cromwellAuthenticationScheme" has been defined.
-  // The currently only available scheme is refresh, which requires a refresh token to be passed upon every workflow submission (see "workflow options" in cromwell documentation).
-  // In that scheme, a client_id and client_secrets must be provided. They must be the ones that have been used to generate the above-mentioned refresh token.
-  // userAuthenticationScheme = "refresh"
-  // refreshTokenAuth = {
-  //   client_id = ""
-  //   client_secret = ""
-  // }
+backend {
+  default = "Local"
+  providers = [
+    {
+      name = "Local"
+      class = "cromwell.engine.backend.local.LocalBackend"
+      config {
+        // Root directory where Cromwell writes job results.  This directory must be
+        // visible and writeable by the Cromwell process as well as the jobs that Cromwell
+        // launches.
+        root: "cromwell-executions"
+
+        // Cromwell makes a link to your input files within <root>/<workflow UUID>/workflow-inputs
+        // The following are strategies used to make those links.  They are ordered.  If one fails
+        // The next one is tried:
+        //
+        // hard-link: attempt to create a hard-link to the file
+        // copy: copy the file
+        // soft-link: create a symbolic link to the file
+        //
+        // NOTE: soft-link will be skipped for Docker jobs
+        localization: [
+          "hard-link", "soft-link", "copy"
+        ]
+      }
+    },
+    {
+      name = "SGE"
+      class = "cromwell.engine.backend.sge.SgeBackend"
+      config {
+        // Root directory where Cromwell writes job results.  This directory must be
+        // visible and writeable by the Cromwell process as well as the jobs that Cromwell
+        // launches.
+        root: "cromwell-executions"
+
+        // Cromwell makes a link to your input files within <root>/<workflow UUID>/workflow-inputs
+        // The following are strategies used to make those links.  They are ordered.  If one fails
+        // The next one is tried:
+        //
+        // hard-link: attempt to create a hard-link to the file
+        // copy: copy the file
+        // soft-link: create a symbolic link to the file
+        //
+        // NOTE: soft-link will be skipped for Docker jobs
+        localization: [
+          "hard-link", "soft-link", "copy"
+        ]
+      }
+    },
+    {
+      name = "JES"
+      class = "cromwell.engine.backend.jes.JesBackend"
+      config {
+        // Google project
+        project = "my-cromwell-workflows"
+
+        // Base bucket for workflow executions
+        baseExecutionBucket = "gs://my-cromwell-workflows-bucket"
```

Comment:
I think this is the function equivalent of "root" in the two other backend configs, if we're going to break config retro-compatibility anyway maybe it's a good time to harmonize the name ?